docs/index.md

```diff
@@ -1,19 +1,20 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: EdRecord
 ---
 
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2122S1-CS2103-W14-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103-W14-3/tp/actions)
 [![codecov](https://codecov.io/gh/AY2122S1-CS2103-W14-3/tp/branch/master/graph/badge.svg?token=5QC0BG91OQ)](https://codecov.io/gh/AY2122S1-CS2103-W14-3/tp)
 
 ![Ui](images/Ui.png)
 
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+**EdRecord is a desktop application for keeping track of your students.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
```

Comment:
Would it be better to sync with the current description we have: 
https://github.com/nus-cs2103-AY2122S1/tp/pull/11#issue-990816195

Reply:
Adapted to your comment!